superset/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN microdnf update \
         gevent \
         psycopg2-binary \
         statsd \
+        pydruid \
```

Comment:
Just a hint: I couldn't find `pydruid` in the constraints.txts. So it seems to me we don't install a fixed version

Reply:
The constraints contain only the libraries which are non-optional dependencies of Superset. This is necessary because otherwise Superset cannot be built or run. All additional libraries are not contained. If adding these libraries then it would make sense to add all transitive dependencies, too.

As the system packages are upgraded to contain the latest bug fixes, I thought it would be a good idea to not constrain the Python packages either if it is not necessary.